orangecontrib/text/widgets/owscraper.py

```diff
@@ -0,0 +1,160 @@
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from Orange.data import Table, StringVariable
+from newspaper import Article
+import tldextract
+from datetime import datetime
+from PyQt4 import QtCore
+
+ARTICLE_TEXT_FIELDS=["Article","Author","Date","Location","Title","URL"]
+
+
+class OWScraper(OWWidget):
+
+
+	# Basic widget info
+    name = "Article Scraper"
+    description = "gets article, title, author, date etc. from the URL"
+    icon = "icons/Scraper.svg"
+    priority = 100
+    category="text"
+
+    # Input/output
+    inputs = [("Article_Links", Table, "set_data")]
```

Comment:
In general, we do not have `_` in names like `Article_Links`. Also, if this is just a regular `Orange.data.Table` then it should be named `Table` rather than `Article_Links`.